homeassistant/components/netatmo/data_handler.py

```diff
@@ -107,6 +107,10 @@ async def handle_event(self, event):
             _LOGGER.info("%s webhook successfully registered", MANUFACTURER)
             self._webhook = True
 
+        elif event["data"]["push_type"] == "webhook_deactivation":
+            _LOGGER.info("%s webhook unregistered", MANUFACTURER)
+            self._webhook = False
```

Comment:
You should dispatch a signal when this happens so availability can be updated.

Reply:
Yes, that is done in `__init__.py L123` https://github.com/home-assistant/core/blob/6d04f3beb70413830bbd69a592527f5397b2c9bd/homeassistant/components/netatmo/__init__.py#L123-L127

Reply:
Isn't that weird that there are 2 places that handle webhooks?

Reply:
Each platform is listening for events that it cares about. But I'll think about how this could be centralized.